dali/kernels/test/tensor_test_utils.h

```diff
@@ -160,7 +160,13 @@ struct EqualEps {
   double eps = 1e-6;
 };
 
-
+/**
+ * Functor for comparing floating point types.
+ * Performs ULP comparision.
+ * Verifies using Saturation Cast
+ *
+ * XXX: Be aware, that when using this functor in `Check()`, order of arguments matter!
```

Comment:
```suggestion
 * XXX: Be aware, that when using this functor in `Check()`, order of arguments matters!
```
BTW - what is XXX: ? If it's not a Doxygen tag, put this in `@remarks` section

Reply:
LMGIFY https://stackoverflow.com/a/7284583